### ISSUE TITLE
Attempting to parse an unsupported color function "oklch"

### ISSUE THREAD

**Issue by keanallen** [johnnywang1994/jsPDF-html2canvas#37]
I got an error while trying to run this plugin in vue3 (inertiajs).
```
color.ts:15 Uncaught (in promise) Error: Attempting to parse an unsupported color function "oklch"
    at Object.parse (color.ts:15:23)
    at parseColor (color.ts:156:11)
    at parseBackgroundColor (index.ts:155:11)
    at index.ts:103:29
    at step (tslib.es6.js:102:23)
    at Object.next (tslib.es6.js:83:53)
    at fulfilled (tslib.es6.js:73:58)
```

**Comment by johnnywang1994** [comment]
Hi, thank you for reporting the issue, it would be great to share the `package.json` for me, so that I could help to locate the issue, were you using "Webpack" or "Vite" or any other bundler to run?

**Comment by yorickshan** [comment]
Hi there, I have committed a PR to fix this issue #43 @johnnywang1994
